frontend/src/index.tsx

```diff
@@ -50,7 +50,7 @@ const authLink = setContext(async (_, { headers }) => {
         { withCredentials: true },
       );
 
-      const accessToken: string = data.data.refresh;
+      const accessToken: string = data?.refresh;
```

Comment:
I found that `data?.refresh` returns undefined while `data.data.refresh` returns the accessToken correctly. It looks weird but I think this section is fine as I am getting the "not authenticated" error without these changes.